doc/index.rst

```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
     symfonycasts_tailwind:
         config_file: 'tailwind.config.js'
 
+If you include any other files containing classes to be detected outside the 
+default directories, like the `form themes provided with the Twig Bridge component <https://github.com/symfony/symfony/blob/HEAD/src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig/>`_, 
+untouched from the ``vendor`` directory, for instance, after changing your Twig configuration like this:
```

Comment:
```suggestion
 from the untouched ``vendor/`` directory, after changing your Twig configuration like this:
```

Reply:
My English might be deficient here: I thought the first one means "didn't copy/paste it in your project from vendor".

Second one seems to mean "not altering the vendor directory", which is by definition.